_playwright-tests/UI/PopularRepo.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { describe } from 'node:test';
+import { test, type Page } from '@playwright/test';
+import { navigateToRepositories } from './helpers/navHelpers';
+import { deleteAllRepos } from './helpers/deleteRepositories';
+import { closePopupsIfExist } from './helpers/helpers';
+
+describe('Popular Repositories', () => {
+  test('Clean - Delete any current repos that exist', async ({ page }) => {
+    await deleteAllRepos(page);
+  });
+
+  test('Navigate to Popular Repos tab, assert the Add button is displayed', async ({ page }) => {
+    await navigateToRepositories(page);
+    await closePopupsIfExist(page);
+
+    const AddMenuButton = page.locator('[data-ouia-component-id="add_checked_repos-with-snap"]');
```

Comment:
For this you can use `getByTestId` - see https://github.com/content-services/content-sources-frontend/pull/424/commits/d3b785345ce84f4d65860346581829a97fc5bc47

Maybe we should add this commit separately to enable others to use it

Reply:
Also, wouldn't there by selector that describes it better? E.g. with text or so? Now it is not clear (to me :) ) what button it actually targets.